Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1211,11 +1211,24 @@ void MediaPlayerPrivateGStreamer::loadingFailed(MediaPlayer::NetworkState networ
 
 GstElement* MediaPlayerPrivateGStreamer::createAudioSink()
 {
+    GstElement* audioSink;
+
+#if PLATFORM(BCM_NEXUS)
+    audioSink = makeGStreamerElement("brcmaudiosink", nullptr);
```

Comment:
This should not be needed. `autoaudiosink` should be able to create this... thing if it's properly ranked and registered as an audio sink.

Reply:
I think they want to ensure you're using that sink only and if it is not present, you crash.

Reply:
Do you think it would make sense to do the same scheme as for the video one? An env var?

Reply:
Having your audio sink selected by autoaudiosink is not rocket science.
For video we can't really use autovideosink, that's why i suggested an env var for that.

Reply:
It looks like we can go with auto plugged.